ipaplatform/redhat/authconfig.py

```diff
@@ -3,6 +3,7 @@
 #          Tomas Babej <tbabej@redhat.com>
 #
 # Copyright (C) 2007-2014  Red Hat
+# Copyright (C) 2016  FreeIPA Contributors
```

Comment:
I don't think this is how the license should look. I'd replace the entire wall of text with just the thre lines with FreeIPA Contributors, but @stlaz would probably know for sure.

Anyway, this change should not be a part of the commit that addresses 5244. Either do not touch the license information, or update to it to the current format in a separate commit.